src/components/Members.tsx

```diff
@@ -3,17 +3,21 @@ import styles from './Members.module.css';
 
 import MemberCard from "./MemberCard";
 
-type Member = {
```

Comment:
I'm thinking about making these types more granular

For example:
- email must begin with mailto: followed by the email address
- bio should be a string no longer than 120 characters
- linkedin and github should begin with their domain